packages/back-end/src/jobs/updateExperimentResults.ts

```diff
@@ -39,7 +43,7 @@ export default async function (agenda: Agenda) {
 
     // New way, based on dynamic schedules
     const experimentIds = (
-      await ExperimentModel.find(
+      await getExperimentsByQuery(
```

Comment:
Same here - create a dedicated model function instead of having a Mongo query here